test/complexNames.wasm.fromBinary

```diff
@@ -0,0 +1,11 @@
+(module
+ (type $0 (func))
+ (export "$foo\20\28.bar\29" (func $1))
+ (func $foo\20\28.bar\29 (; 0 ;) (type $0)
+  (nop)
+ )
+ (func $1 (; 1 ;) (type $0)
```

Comment:
does the binary not have a names section name for this function?

Reply:
it has -- wasm-as exported name as "1"